pkg/webui/components/map/widget/index.js

```diff
@@ -42,10 +42,19 @@ export default class MapWidget extends React.Component {
 
     const leafletConfig = {
       zoomControl: false,
+      zoom: 7,
+      minZoom: 1,
     }
+    const mapCenter = [markers[0].position.latitude, markers[0].position.longitude]
```

Comment:
```suggestion
    const mapCenter = Boolean(markers) && markers.length !== 0 ? [markers.length && markers[0].position.latitude, markers[0].position.longitude] : defaultLocation
```
This will crash the console if there is no location set. Please fix and double check that there is no issues with different location configurations.